src/views/employee.hbs

```diff
@@ -0,0 +1,99 @@
+ <section class="employee-section">
+    <div class="title">
+        <h2 class="greet">أهلا في مركز القطان للدعم الفني </h2>
```

Comment:
Qattan center is not a tech support center.

Should say something like: "tech support department of al qattan"